web/.vscode/settings.example.json

```diff
@@ -2,7 +2,7 @@
   "prettier.enable": false,
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
> Code Actions' behavior on save in the editor has been modified from Booleans to an enum with the following options:
`always`: Code Actions with always be run on any kind of save (autosave when exiting or changing focus, explicit save, etc).
`explicit`: Code Actions will be run only when explicitly saving.
`never`: Code Actions will never be run on save.

See https://github.com/microsoft/vscode/issues/163920#issuecomment-1721561959